app/controllers/derby_events_controller.rb

```diff
@@ -95,6 +95,7 @@ def format_date_for_db(form_date)
     def derby_event_params
       event_params = params.require(:derby_event).permit(:name, :start_date, :start_time, :end_date, :end_time, :venue, :city, :state, :postal_code, :country, :host, :website, :event_contact, :cost, :mrda, :wftda, :jrda, :made, :usars, :rdcl, :female, :male, :coed, :preregistration, :spectators, :general_info, :submission_contact, :approved, :deleted)
       [:start_date, :end_date].each do |date_field|
+
```

Comment:
This is great - exactly what controllers should be doing, formatting form input for the model